src/cl/cl_launcher.star

```diff
@@ -171,30 +153,16 @@ def launch(
                 plan,
                 cl_launcher,
                 cl_service_name,
-                participant.cl_image,
-                participant.cl_log_level,
+                participant,
                 global_log_level,
                 cl_context_BOOTNODE,
                 el_context,
                 full_name,
                 new_cl_node_validator_keystores,
-                cl_min_cpu,
-                cl_max_cpu,
```

Comment:
im confused, why is all this being removed?

Reply:
because its coming from participant.cl_min_cpu